pyproject.toml

```diff
@@ -22,13 +22,13 @@ click = "^8.1"
 tabulate = { version = "^0.9", optional = true }
 pandas = { version =  "^2.0", optional = true }
 bokeh = { version =  "^3", optional = true }
+pytest = { version = "^7.4.0", optional = true }
 
 [tool.poetry.extras]
-scripts = ["tabulate", "pandas"]
+scripts = ["tabulate", "pandas", "pytest"]
 plotting = ["bokeh"]
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^7.4.0"
```

Comment:
I don't understand why it is no longer in the `dev` dependencies. We now have testing for the package right, so it probably belongs here still